apps/docs/components/playground-utils/files.js

```diff
@@ -0,0 +1,393 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+export const files = {
+  public: {
+    directory: {
+      'index.html': {
+        file: {
+          contents: `
```

Comment:
The structure of this object is kind of difficult to understand. Could you put the `contents` strings into individual files?

Reply:
@Jta26 this isn't super easy to do with web containers so this will proabably need to remain. This is also not what this PR does.

Reply:
No, I mean just split this big JS object up into separate files so it's easier to understand, or even just put the `contents` into a separate variable. I don't mean turn the string into a file and somehow import it and use it as a string.